Part1/Proj1/ViewController.swift

```diff
@@ -7,13 +7,90 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+func isIPhone() -> Bool {
+return UIDevice.current.userInterfaceIdiom == .phone
+}
+class someVC:
```

Comment:
Naming Conventions:
   - I noticed that the class name `class someVC` does not adhere to Swift naming conventions. To make the code more readable and aligned with best practices, please rename it to `class SomeVC`. Consistency in naming conventions is crucial for maintainability.